torch/_inductor/codegen/triton.py

```diff
@@ -1202,6 +1204,29 @@ def __call__(self):
             def _new_line(self, line):
                 return IndirectAssertLine(line, self.var, self.mask, self.size_map)
 
+        if var.bounds.lower < 0:
+            str_size = self.index_to_str(size)
+            new_bounds = ValueRanges.unknown()
+            if var.bounds != ValueRanges.unknown():
+                # Take the negative part of the bound and add size to it
+                # Then take union of that and the positive part
+                # This is a tighter bound than that of a generic ops.where, as we have info on the conde
```

Comment:
nit: conde -> code ;-)

Reply:
cond* (as in the condition from the `where`) but yep :D